src/framework/uicomponents/qml/Muse/UiComponents/StyledDropdown.qml

```diff
@@ -107,6 +107,7 @@ Item {
         id: mainItem
 
         anchors.fill: parent
+        label.anchors.rightMargin: 30
```

Comment:
In this case, I see that we are managing the component content from the outside - we usually don't do that.

I would suggest the following:
1. move `iconItem` from `StyledDropdown.qml` to `DropdownItem.qml`. The icon is related to the item, so why shouldn't it be there?
2. fix anchors for `labelItem` and `iconItem` after moving
3. add `property alias dropIcon: mainItem.icon` to `StyledDropdown.qml`. I see that we are using the icon somehow outside of StyledDropdown, so we need to keep this option

Reply:
(For context: previuosly, `DropdownItem` was also used as the `delegate` inside `StyledDropdownView`. In that place, there should of course not be an icon, so therefore the icon is not in `DropdownItem`. But (as I just found) `DropdownItem` is no longer used in `StyledDropdownView`. 

That raises the question whether `DropdownItem` still needs to be a separate component at all. I think it could just as well be integrated into `StyledDropdown.qml`.)

Reply:
There we go. Observations:

1. It seems to me that the `selected` and `insideDropdownList` properties are not used (are always `false`), respectively state `"FOCUSED_INSIDE_DROPDOWN"` and `"SELECTED"` will never fire.

2. On `labelItem` I am setting `anchors.left`, `anchors.top`, `anchors.bottom` and `anchors.right` and `anchors.right` is set to the drop icon's `left`. Interestingly, attempting to set `anchors.right: dropIconItem.left - 6` resulted in the text being very wide and exceeding the drop-down width.  If someone can give me an explanation, that'd be great. I've set `anchors.rightMargin: 6` instead.